### ISSUE TITLE
(Optionally) Warn user when an AUR package is orphan (no maintainer)

### ISSUE THREAD

**Issue by ArchangeGabriel** [rmarquis/pacaur#617]
###### Description

On `pacaur -Syu`, I’d like to have a warning on orphaned AUR package. Part of the packages I maintain currently are because of yaourt printing this warning. ;)

###### Output

That would be something like:

`pacaur -Syu`

```
<skippedlines>
:: python2-dumpy isn’t present in the AUR -- skipped
:: gstreamer0.10-good-morituri is orphaned in the AUR
:: gstreamer0.10-good-plugins-morituri is orphaned in the AUR
```

**Comment by rmarquis** [comment]
Implemented in 0421b5321975f53bacfbf18783b70ee9bb28107a.

**Comment by ArchangeGabriel** [comment]
That was fast, thanks a lot!
